decentraland/screens/EthereumSignUpScreen.js

```diff
@@ -26,7 +30,11 @@ const {
 export class EthereumSignUpScreen extends React.Component {
   _onboarding = async () => {
     try {
-      const { setAccount, setAccountCreationStatus } = this.props;
+      const {
+        setAccount,
+        setAccountCreationStatus,
+        setAccountCreationCurrentAction,
+      } = this.props;
 
       // The pattern for each step is:
       // 1. alert with good info as soon as it's true
```

Comment:
We should add the recommendation of where the "setting action" action (two actions haha) belongs int his flow. It seems as early as possible is the way to go.

Reply:
Comment updated

Reply:
Thanks. Resolving this